packages/create-toolpad-app/src/core.ts

```diff
@@ -0,0 +1,58 @@
+import chalk from 'chalk';
+import { execa } from 'execa';
+import type { GenerateProjectOptions } from './types';
+import generateProject from './generateProject';
+import writeFiles from './writeFiles';
+import { findCtaPackageJson, getPackageManager } from './package';
+
+export async function scaffoldCoreProject(options: GenerateProjectOptions): Promise<void> {
+  // eslint-disable-next-line no-console
+  console.log();
+  // eslint-disable-next-line no-console
+  console.log(
+    `${chalk.cyan('info')} - Creating Toolpad Core project in ${chalk.cyan(options.absolutePath)}`,
+  );
+  // eslint-disable-next-line no-console
+  console.log();
+
+  const pkg = await findCtaPackageJson();
```

Comment:
Better to remove this call and pass the version as an option.

Reply:
If we remove the call, how would we set the default value of the version in case nothing is passed in the version option?

Reply:
If there is no sensible default, then you make the option mandatory